ldp/data_structures.py

```diff
@@ -121,7 +122,12 @@ def from_jsonl(cls, filename: str | os.PathLike) -> Self:
             reader = iter(f)
             traj = cls(traj_id=json.loads(next(reader)))
             for json_line in reader:
-                traj.steps.append(Transition(**json.loads(json_line)))
+                data = json.loads(json_line)
+                # logprob may have been serialized, but cnanot be passed to
+                # OpResult, so remove it here.
+                with suppress(KeyError):
+                    data["action"].pop("logprob")
```

Comment:
Why can `logprob` not be passed to `OpResult`'s action?

Also, typo in "cannot"

Reply:
`logprob` is a property that we extract from the OpCtx, not an attribute of the OpResult, so there's no way to set it on OpResult deserialization - the correct thing to do is also hydrate the OpCtx from the backend.